test/screenshot/text-field/index.js

```diff
@@ -90,10 +109,31 @@ const textFields = variants.map((variant) => {
   });
 });
 
+const textareaVariants = [
+  {},
+  {fullWidth: true},
+];
+
+const textareaFields = textareaVariants.map((variant) => {
+  return denseMap.map((dense) => {
+    return rtlMap.map((isRtl) => {
+      return disabledMap.map((disabled) => {
+        return helperTextMap.map((helperText) => {
+          const props = Object.assign({}, variant, dense, disabled, helperText, isRtl);
+          const key = JSON.stringify(props);
+          return <TestTextareaField {...props} key={key} id={key} />;
```

Comment:
I think you can just use `<TestField textarea ...>` here, no need to create `TestTextareaField` :)